seaclouds-catalog.bom

```diff
@@ -254,6 +258,8 @@ brooklyn.catalog:
               planner.port: $brooklyn:component("planner").config("planner.config.port")
             install.latch: $brooklyn:component("planner").attributeWhenReady("service.isUp")
           - type: seaclouds-planner
+            brooklyn.config: 
+               planner.config.optimizer.url: $brooklyn:formatString("http://%s:%s", component("optimizer-service").attributeWhenReady("host.address"), component("optimizer-service").attributeWhenReady("http.port"))
```

Comment:
Thank you Adrián!!
 
I'm waiting for some comments/advices regarding the cloud deployment. My doubt was: should I place the optimizer in a dedicated VM or colocated with other modules?